### PR TITLE
Implement prototype audio recorder.

### PR DESCRIPTION
**Pull request by max-vogler**
See #205.

### CODE REVIEW

**Comment by hoch** [comment]
Looks great! Can we have minimum comments on the JS file? Are you using a linter?

**Review by hoch — COMMENTED** [review]
I also suggest adding a very simple README.md to explain what it does and how it is done.

**Comment by max-vogler** [comment]
> Looks great! Can we have minimum comments on the JS file? Are you using a linter?

Added comments. All code is formatted with prettier.io.

> I also suggest adding a very simple README.md to explain what it does and how it is done.

Added README.

File: audio-recorder/app.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+"use strict";
```
[the remark this answers is not in the document]
Comment (max-vogler) [reply audio-recorder/app.js:1]: I use prettier.io which defaults to double-quotes. Shall I add a configuration file to override it, or leave it at double quotes?

File: audio-recorder/app.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    const volume = Math.max(...dataArray) / 255;
+
+    if (mediaRecorder.state !== "recording") {
+      recordOutlineEl.style.boxShadow = "none";
+      return; // Stop drawing loop.
+    }
+
+    recordOutlineEl.style.boxShadow = `0 0 0 ${
+      volume * 20
+    }px rgba(0, 0, 0, 0.2)`;
```
[the remark this answers is not in the document]
Comment (max-vogler) [reply audio-recorder/app.js]: Yes. I extracted `volume * 20` into a variable and now Prettier leaves the format string in one line.

File: audio-recorder/app.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  const audioCtx = new AudioContext();
+  const source = audioCtx.createMediaStreamSource(stream);
+  const analyser = audioCtx.createAnalyser();
+  analyser.fftSize = 32;
+  const bufferLength = analyser.frequencyBinCount;
+  const dataArray = new Uint8Array(bufferLength);
+
+  source.connect(analyser);
+
+  let x = -1;
```
[the remark this answers is not in the document]
Comment (max-vogler) [reply audio-recorder/app.js]: Added.

**Comment by hoch** [comment]
Perhaps I should've shared this with you earlier. FYI:
https://github.com/GoogleChrome/samples#style--linting--ci

For future contribution, let us follow the guideline. This PR looks great as is.

**Review by hoch — APPROVED** [review]

**Pull request merged**